packages/test-runner-module-mocking/test/moduleMockingPlugin.test.ts

```diff
@@ -20,6 +20,14 @@ describe('moduleMockingPlugin', function test() {
     });
   });
 
+  it('can intercept server relative modules with stub', async () => {
```

Comment:
All tests are using stubs. :-)
Example: https://github.com/modernweb-dev/web/blob/2d933bcf7d8d4a1b2f67dcd6413985bb4c692356/packages/test-runner-module-mocking/test/fixtures/server-relative/browser-test.js#L16

We don't want to test all different stubbing libraries.

So, you can remove this extra test.